src/server/page_server.cpp

```diff
@@ -399,10 +478,15 @@ page_server::set_passive_tran_server_connection (cubcomm::channel &&chn)
   assert (is_page_server ());
 
   chn.set_channel_name ("PTS_PS_comm");
+
+  const auto channel_id = chn.get_channel_id ();
```

Comment:
@wrlawodms I think this PR can be merged only if a solution for the unique channel id is found.

I suggest to take that as a separate issue using either my suggestion from https://github.com/CUBRID/cubrid/pull/3837#discussion_r989243338 or another solution.

And then come back and test&merge this one.

We can go forward with the current code with only one PTS, but the test infrastructure that we have in place for LETS tests uses 2 PTS's. And a few tests will crash.

Or another option that involves disabling this functionality in the `scalability_dev` branch.

 Your choice.

Reply:
It seems better to get the unique channel id issue done first. I'll create an issue for it.